native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/NativeBuildMojo.java

```diff
@@ -155,6 +191,108 @@ public void execute() throws MojoExecutionException {
         }
     }
 
+    private void configureMetadataRepository() {
+        if (isMetadataRepositoryEnabled()) {
+            Path repoPath = null;
+            if (metadataRepositoryConfiguration.getVersion() != null) {
+                getLog().warn("The official JVM reachability metadata repository is not released yet. Only local repositories are supported");
```

Comment:
This means that we'd have to release a new version once the repo is out. The Gradle plugin is more optimistic and assumes this will work.

Reply:
What the Gradle plugin does is not working today, and will not work in the future either, since the final URL will be different than what was hardcoded here: https://github.com/graalvm/native-build-tools/blob/master/native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/NativeImagePlugin.java#L413-L413

If I'm able to test it (see #241), I'll add support in this PR for remote repositories URLs, which will allow users to use the official repository ZIP URL once it's released.

Reply:
Well yes but you can override the URL in any case.

Reply:
Correct, that's what I want to implement. Provide a URL explicitly. And that is not where you added the comment.

The "automatic URL" thing (by just providing the version) is even purposely not documented.

Reply:
I have implemented remote repositories support and tested them by spawning a local HTTP server.